source/slang/slang-ir.h

```diff
@@ -564,6 +564,7 @@ enum class IRTypeLayoutRuleName
     Scalar = Natural,
     Std430,
     Std140,
+    ConstantBuffer,
```

Comment:
Is this LegacyD3DConstantBuffer ?

Reply:
Yes, `ConstantBuffer` meant the `cbuffer` block `ConstantBuffer`.

Reply:
Let's name it D3DConstantBuffer to be precise.